main/res/values-de/strings.xml

```diff
@@ -172,7 +172,7 @@
   <string name="err_application_no">c:geo konnte keine passende Anwendung finden.</string>
   <string name="err_auth_initialize">c:geo konnte die Autorisierung nicht starten.</string>
   <string name="err_auth_process">Autorisierung fehlgeschlagen.</string>
-  <string name="err_cannot_log_visit">c:geo hat nicht genügend Informationen um den Besuch zu loggen. Bitte über die Cache-Details versuchen.</string>
+  <string name="err_cannot_log_visit">c:geo hat nicht genügend Informationen, um den Besuch zu loggen. Bitte über die Cache-Details versuchen.</string>
```

Comment:
Mist, fast immer ist es bei solchen Infinitivgruppen optional und dann kommt in Regel 116 eine Ausnahme.

http://www.duden.de/sprachwissen/rechtschreibregeln/komma#K114

Das stammt aber auch aus der Änderungen der Rechtschreibung im Jahr 2006 gegenüber 2004, während ich in der Schule die Reform von 1996 gelernt hatte. Ist alles viel zu kompliziert.